contrib/devtools/Makefile

```diff
@@ -52,17 +52,28 @@ tools: statik runsim golangci-lint
 
 golangci-lint: $(GOLANGCI_LINT)
 $(GOLANGCI_LINT): $(mkfile_dir)/install-golangci-lint.sh
-	bash $(mkfile_dir)/install-golangci-lint.sh $(TOOLS_DESTDIR) $(GOLANGCI_LINT_VERSION) $(GOLANGCI_LINT_HASHSUM)
+	@echo "Installing golangci-lint..."
+	@bash $(mkfile_dir)/install-golangci-lint.sh $(TOOLS_DESTDIR) $(GOLANGCI_LINT_VERSION) $(GOLANGCI_LINT_HASHSUM)
 
+# Install the runsim binary with a temporary workaround of entering an outside
+# directory as the "go get" command ignores the -mod option and will polute the
+# go.{mod, sum} files.
+# 
+# ref: https://github.com/golang/go/issues/30515
 statik: $(STATIK)
 $(STATIK):
-	@echo "Get statik"
-	@go get github.com/rakyll/statik
+	@echo "Installing statik..."
+	@(cd /tmp && go get github.com/rakyll/statik@v0.1.6)
 
+# Install the runsim binary with a temporary workaround of entering an outside
+# directory as the "go get" command ignores the -mod option and will polute the
+# go.{mod, sum} files.
+# 
+# ref: https://github.com/golang/go/issues/30515
 runsim: $(RUNSIM)
 $(RUNSIM):
-	@echo "Get runsim"
-	@go get github.com/cosmos/tools/cmd/runsim
+	@echo "Installing runsim..."
+	@(cd /tmp && go get github.com/cosmos/tools/cmd/runsim@v1.0.0)
```

Comment:
I suspect that this wouldn't work on Windows. But maybe we don't care that much

Reply:
Do you have alternatives @alessio? If so feel free to open a PR.

Reply:
I looked into it. GNU Make's manual mentions TMPDIR in Directory Variables (https://www.gnu.org/software/make/manual/html_node/Directory-Variables.html) though Make does not provide guarantee that such directory is instantiated upon execution.

I'll come up with something and open a PR